scripts/fetch_vision.sh

```diff
@@ -23,11 +23,10 @@ apt-get -o Dpkg::Use-Pty=0 update -qq  || echo " RC${?} Got an error on update??
 apt-get -o Dpkg::Use-Pty=0 install -qq aria2
 echo "Downloading to ${RAMDISK}..."
 pushd $RAMDISK
-echo "Fetching  image tarball..."
-#Use xargs to pass the stdout of the signing script to aria2c as an arugment
-python3 $BASEDIR/sign.py --url $URLPAIVIMAGES | xargs -t aria2c -q -s160 -x16 $URLPAIVIMAGES
-echo "Fetching deb"
-python3 $BASEDIR/sign.py --url $URLPAIVDEB | xargs -t aria2c -q $URLPAIVDEB
+echo "Fetching image tarball from ${URLPAIVIMAGES}"
```

Comment:
I think the signing stuff is still useful here; i'd like to bring it back as a configurable thing based on whether the signing keys are set or not but this is fine.